models/src/main/thrift/content/v1.thrift

```diff
@@ -1553,7 +1553,10 @@ struct Content {
 
     25: optional string pillarName
 
-    26: optional Platform.Platforms platforms
+    /*
+     * Where has this piece of content been published?
+     */
+    26: optional list<Platform.PublicationInstance> publicationInstances
```

Comment:
My fear is that this will make querying slow and introduce much complexity in the code. The way I see it is that we have three options:

- **One item per article in the index**, with all platform-specific data distilled into the `PublicationInstance`s
  - pros: one ID to rule them all, backwards-compat is just adding a filter in Concierge, no downstream change to clients
  - cons: implies the content is the same everywhere; not necessarily bad, esp. if we follow-up with platform-specific markup.

- **One item per platform per article in the index**, i.e. each item in the index exists for a single platform
  - pros: more flexibility with schema evolution in the future instead of shoving everything into `PublicationInstance`s, the content of each item (metadata and all) can be fully customised per platform
  - cons: the ID becomes a tuple `(path, platform)` that needs (de-)normalisation, I'm not sure we need that level of flexibility

- **One item per list of platforms in the index**, i.e. one item can be use for several platforms
  - pros: less redundancy than option 2 but same flexibility with the content
  - cons: need for a new ID, risk of introducing inconsistency (duplicates) pushes the complexity down into CAPI

With option 2 and 3, querying by ID will need additional filters and denormalising. Just flagging this out, if we can live with option 1 at the beginning it seems to be less effort.